### ISSUE TITLE
Find correlations between personal preferences and song metadata

### ISSUE THREAD

**Issue by epistimi4** [epistimi4/emotion_based_music_recomendation#9]
Make a new python program to investigate the correlations between personal preferences and song metadata

**Comment by matsarou** [comment]
Saying that we want to correlate the mood taken from the personal metadata, with the mood of the songs in spotify, I am wondering about two things:
1)Can we set as personal mood of a song, the field <emotional_state> of the person, or should the mood derive from the combination of the fields <emotional_state> and the other fields of the csv file(age,gender etc.)?
2)We intend to create a mood value for each song, by clustering based on their spotify metadata(audio features) and then labeling the clusters and their songs. In order to do this we must first find the most representative features of each cluster of songs. This can be done in two ways:
- manually, by viewing the content of each cluster of songs 
-  automatically(with supervised learning). 
@tyiannak what is your opinion? Can we do it manually, in case we don't manage to implement the automatic way?

**Comment by tyiannak** [comment]
The goal of this task is to measure the correlation between song metadata (not audio features of the songs, that is a separate task I guess) and user preferences. One way to solve this is to see this task as a task of predicting the user's state (context, mood, whatever) based on the (aggregated) song metadata. In other words, to answer the question: based on these 10 songs with these 10 metadata entries, can one predict the user's mood and / or context? (e.g. stressed + at the gym)?

**Comment by epistimi4** [comment]
is blocked by #15

**Comment by matsarou** [comment]
@epistimi4 @athinabikaki @natalitsa 
I made some progress, with a sample preprocessing code I made. I believe that we can use the same classifier for both youtube and spotify data.
We will have to call him with a change on one parameter discriminator. 

I will push the code today and we will discuss it again later.

**Comment by matsarou** [comment]
I created a separate issue for the classification  #18.
This issue is now only for finding correlations between metadata. 
@athinabikaki is on it.

**Comment by athinabikaki** [comment]
clustering algorithms were used to correlate context, emotion and Spotify metadata
